lizmap/modules/lizmap/classes/lizmapOGCRequest.class.php

```diff
@@ -20,17 +20,75 @@ class lizmapOGCRequest
 
     protected $params;
 
+    protected $requestXml;
+
     protected $services;
 
     protected $tplExceptions;
 
+    static public function build($project, $params, $requestXml = Null)
+    {
+        $service = Null;
+        $request = Null;
+
+        // Check request XML
+        if ($requestXml && substr(trim($requestXml), 0, 1) == '<') {
```

Comment:
Should we validate XML this way ?

Reply:
I didn't find an other way ;-) because `$requestXml = file('php://input');` returns POST DATA and not only XML POST DATA